tests/ops/test_qubit_ops.py

```diff
@@ -873,6 +912,13 @@ def test_qubit_unitary_not_matrix_exception(self, U):
         with pytest.raises(ValueError, match="must be a square matrix"):
             qml.QubitUnitary(U, wires=0).matrix
 
+    def test_iswap_eigenval(self):
+        """Tests that the ISWAP eigenvalue matches the numpy eigenvalues of the ISWAP matrix"""
+        op = qml.ISWAP(wires=[0, 1])
+        exp = np.linalg.eigvals(op.matrix)
+        res = op.eigvals
+        assert np.allclose(res, np.sort_complex(exp))
```

Comment:
Could we get away without a call to `np.sort_complex`? Locally the order seems to match the one defined for the operation.